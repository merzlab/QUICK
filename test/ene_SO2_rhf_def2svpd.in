HF  BASIS=def2-svpd cutoff=1.0e-9 denserms=1.0e-6  zmake ENERGY DIPOLE CHARGE=0

S         -1.29718        0.85107        0.08345
O          0.13448        0.85245       -0.13524
O         -1.56820        0.86002       -1.33920


