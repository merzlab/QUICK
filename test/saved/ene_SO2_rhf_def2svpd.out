  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Thu Mar  4 18:27:00 2021
| INPUT FILE :    ene_SO2_rhf_def2svpd.in
| OUTPUT FILE:    ene_SO2_rhf_def2svpd.out
| DATE FILE  :    ene_SO2_rhf_def2svpd.dat
| BASIS SET PATH: /home/mmanathunga/fix109/basis

 @ Read Job And Atom

  KEYWORD=HF  BASIS=DEF2-SVPD CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE ENERGY DIPOLE CHARGE=0                                                                                                                         
 ============== JOB CARD =============
 METHOD = HATREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SVPD,  TYPE = CARTESIAN
| BASIS FILE = /home/mmanathunga/fix109/basis/DEF2-SVPD.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = S 
 BASIS FUNCTIONS =   29
 For Atom Kind =    2
 ELEMENT = O 
 BASIS FUNCTIONS =   25

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    3    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    0    NUMBER OF NON-HYDROGEN ATOM =    3
 NUMBER OF ELECTRONS     =   32

 -- INPUT GEOMETRY -- :
    S          -1.2972       0.8511       0.0835
    O           0.1345       0.8525      -0.1352
    O          -1.5682       0.8600      -1.3392

 -- DISTANCE MATRIX -- :
              1           2           3
      1     0.00000
      2     2.73683     0.00000
      3     2.73682     3.94075     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   79
 NSHELL =   29 NPRIM  =  117
 JSHELL =   29 JBASIS =   51


 @ End Reading Molecular Information 


 @ Begin Energy Calculation

                                         SCF ENERGY
------------------------------------------------------------------------------------------------------------------------
NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
------------------------------------------------------------------------------------------------------------------------
|  1   -548.085208893      ------         0.706     1        0.00      0.70      0.00    0.2075E+01  0.3003E-01  0.4617E+00
|  2   -546.764133604  -.132108E+01       0.894     2        0.00      0.89      0.00    0.2631E+00  0.1433E-01  0.3898E+00
|  3   -546.820626122  0.564925E-01       0.892     3        0.00      0.89      0.00    0.2065E+00  0.6324E-02  0.1437E+00
|  4   -546.897454835  0.768287E-01       0.891     4        0.00      0.89      0.00    0.3307E-01  0.1348E-02  0.2035E-01
|  5   -546.900230497  0.277566E-02       0.891     5        0.00      0.89      0.00    0.9096E-02  0.8160E-03  0.1031E-01
|  6   -546.900989143  0.758647E-03       0.890     6        0.00      0.89      0.00    0.6656E-02  0.4088E-03  0.4564E-02
|  7   -546.901225040  0.235897E-03       0.891     7        0.00      0.89      0.00    0.2563E-02  0.3762E-03  0.5291E-02
|  8   -546.901331004  0.105964E-03       0.837     8        0.00      0.83      0.00    0.1344E-02  0.2438E-03  0.4174E-02
|  9   -546.901365913  0.349091E-04       0.755     9        0.00      0.75      0.00    0.7287E-03  0.1534E-03  0.2831E-02
| 10   -546.901374962  0.904850E-05       0.758    10        0.00      0.75      0.00    0.2220E-03  0.4374E-04  0.8110E-03
| 11   -546.901375799  0.837722E-06       0.753     1        0.00      0.75      0.00    0.6849E-04  0.1298E-04  0.2108E-03
| 12   -546.901375883  0.836826E-07       0.757     2        0.00      0.75      0.00    0.2898E-04  0.3821E-05  0.6866E-04
| 13   -546.901375893  0.955070E-08       0.765     3        0.01      0.76      0.00    0.1259E-04  0.1666E-05  0.3077E-04
| 14   -546.901375895  0.200122E-08       0.759     4        0.00      0.75      0.00    0.5214E-05  0.1073E-05  0.1744E-04
| 15   -546.901375895  0.563190E-09       0.754     5        0.00      0.75      0.00    0.2049E-05  0.1231E-06  0.1649E-05
------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  15 CYLCES
| MAX ERROR = 0.204942E-05   RMS CHANGE = 0.123055E-06   MAX CHANGE = 0.164921E-05
 -----------------------------------------------
 ELECTRONIC ENERGY    =  -656.681035509
 CORE_CORE REPULSION  =   109.779659614
 TOTAL ENERGY         =  -546.901375895

 @ End Energy calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      S        0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      O        1.44827  1     0.00000  0      0.00000  0      1    0    0
     3      O        1.44826  1    92.10048  1      0.00000  0      1    2    0

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   S              1.1172             1.3160
   O             -0.5586            -0.6634
   O             -0.5586            -0.6526
   TOTAL        -0.0000            -0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -2.1512   -0.0191    3.0421    3.7259

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =    15.046029000( 54.99%)
| TOTAL SCF TIME      =    12.202829000( 44.60%)
|       TOTAL OP TIME      =    12.134822000( 44.35%)
|             TOTAL 1e TIME      =     0.009969000(  0.04%)
|             TOTAL 2e TIME      =    12.114349000( 44.28%)
|       TOTAL DII TIME      =     0.067340000(  0.25%)
|             TOTAL DIAG TIME    =     0.018076000(  0.07%)
| TOTAL TIME          =    27.361387000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes 27.4 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Thu Mar  4 18:27:27 2021
